src/conf_mode/interfaces_ethernet.py

```diff
@@ -20,6 +20,7 @@
 
 from vyos.base import Warning
 from vyos.config import Config
+from vyos.configdep import set_dependents, call_dependents
```

Comment:
Please use one import per line - this makes it easier `grep` - able for refactorings

```suggestion
from vyos.configdep import set_dependents
from vyos.configdep import call_dependents
```